src/commands/junit/upload.ts

```diff
@@ -97,6 +102,14 @@ export class UploadJUnitXMLCommand extends Command {
       this.config.env = this.env
     }
 
+    if (
+      !this.logs &&
```

Comment:
why do you need to check `!this.logs` ?

Reply:
To make --logs override the env var value so that it can be true even if the env var is not defined